docs/index.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: home
+---
+
+# Gemara: GRC Engineering Model for Automated Risk Assessment
+
+> Pronounced: Juh-MAH-ruh (think 💎)
+
+[![Go Reference](https://pkg.go.dev/badge/github.com/ossf/gemara.svg)](https://pkg.go.dev/github.com/ossf/gemara)
+
+## What is Gemara?
+
+Gemara (The _GRC Engineering Model for Automated Risk Assessment_) provides a logical model to describe the categories of compliance activities, how they interact, and the schemas to enable automated interoperability between them.
```

Comment:
```suggestion
Gemara _(the GRC Engineering Model for Automated Risk Assessment)_ provides a logical model to describe the categories of compliance activities, how they interact, and the schemas to enable automated interoperability between them.
```

This was the only thing on the page that looked odd to me, since it was cutting off some of the lettering